pheweb/serve/data_access/db.py

```diff
@@ -124,7 +125,8 @@ def get_matching_result(self, resultname):
     
     def json_rep(self):
         return {'phenocode':self.phenocode,'phenostring':self.phenostring, 'pval':self.pval, 'beta':self.beta, "maf":self.maf, "maf_case":self.maf_case,
-                "maf_control":self.maf_control, 'matching_results':self.matching_results, 'category':self.category_name, "n_case":self.n_case, "n_control":self.n_control, "n_sample":self.n_sample}
+                "maf_control":self.maf_control, 'matching_results':self.matching_results, 'category':self.category_name, 'category_index': self.category_index,
+                "n_case":self.n_case, "n_control":self.n_control, "n_sample":self.n_sample}
```

Comment:
Would it make sense to use the `self.__dict__`-magic member, which automagically creates a dictionary of the object members? Then there's no need to update the function when updating member variables. The only thing one needs to keep in mind is to have meaningful names for the member variables, because `self.__dict__` returns a dict of form `{'membername1':membervalue1, 'membername2':membervalue2, 'membername3':membervalue3}`
Like so:
```python
    def json_rep(self):
        return self.__dict__

```

Reply:
Hey this is really neat, thanks!